src/ert/shared/net_utils.py

```diff
@@ -70,49 +69,30 @@ def find_available_socket(
 
     See e.g. implementation and comments in EvaluatorServerConfig
     """
-    current_host = custom_host if custom_host is not None else _get_ip_address()
-    current_range = (
-        custom_range if custom_range is not None else range(51820, 51840 + 1)
-    )
+    current_host = host if host is not None else get_ip_address()
 
-    if current_range.start == current_range.stop:
-        ports = list(range(current_range.start, current_range.stop + 1))
+    if port_range.start == port_range.stop:
```

Comment:
This shouldn't happen anymore, right?

Reply:
it won't happen from EnsembleEvaluator, but others could call it like that

Reply:
right, I thought this was in evaluator :)